src/controllers/expensesController.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+const expenseService = require('../services/expensesServices.js');
+const { getById: getUserById } = require('../services/usersServices.js');
+
+const get = async (req, res) => {
+  const expenses = await expenseService
+    .getAll(req.query)
+    .then((exps) => exps.map(expenseService.normalize));
```

Comment:
Inconsistent promise handling: Consider using `async/await` instead of mixing it with `.then()`. This will make the code more readable and maintainable.